src/pytorch_lightning/strategies/deepspeed.py

```diff
@@ -82,7 +82,7 @@ def forward(self, *inputs: Any, **kwargs: Any) -> Any:
 
     def _batch_to(self, batch: Tensor) -> Tensor:
         if torch.is_floating_point(batch):
-            if self.precision == PrecisionType.HALF:
+            if self.precision in (PrecisionType.HALF, PrecisionType.MIXED):
```

Comment:
this is anyways deprecated in another PR and will be merged in bug-fix release, so no need to test this.